glassfish-tck-runner/pom.template.xml

```diff
@@ -42,10 +42,11 @@
         <htmlunit.version>2.9</htmlunit.version>
         <shrinkwrap.version>1.0.1</shrinkwrap.version>
         <minimum.maven.version>3.0.5</minimum.maven.version>
-        <cdi.api.version>3.0.0</cdi.api.version>
-        <cdi.tck.version>3.0.3</cdi.tck.version>
+        <cdi.api.version>4.0.0</cdi.api.version>
+        <cdi.tck.version>4.0.1</cdi.tck.version>
         <httpcomponents.version>4.5.5</httpcomponents.version>
         <arquillian.version>1.7.0.Alpha9</arquillian.version>
```

Comment:
Can we move to the aquillian version to 1.7.0.Alpha10 ? I saw the CDI TCK release documentation saying that this is the version to use

Reply:
```suggestion
        <arquillian.version>1.7.0.Alpha10</arquillian.version>
```